### PR TITLE
Update argument description of web service

### PR DESCRIPTION
**Pull request by zymap**
The web service api documents are lack of some descriptions.

### CODE REVIEW

**Comment by jiazhai** [comment]
run java8 tests

File: pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/BrokersBase.java [hunk 1]
```diff
@@ -90,12 +97,13 @@
     }
 
     @GET
-    @Path("/{cluster}/{broker}/ownedNamespaces")
+    @Path("/{clusterName}/{broker-webserviceurl}/ownedNamespaces")
```
Comment (jiazhai) [inline pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/BrokersBase.java]: Seems other rest endpoints also use cluster/broker for short, Is here a special need to change this?

**Comment by jiazhai** [comment]
over all lgtm. @Jennifer88huang @Anonymitaet would you please also take a look, if you have time?

**Comment by Anonymitaet** [comment]
@jiazhai LGTM.

File: pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/BrokersBase.java (hunk 1 above)
Comment (zymap) [reply pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/BrokersBase.java]: I think `broker-webservice` is necessary. Firstly I think `broker` is broker service url. So I think if using `broker-webservcie` is more clear.

**Comment by jiazhai** [comment]
run java8 tests

**Comment by jiazhai** [comment]
run java8 tests

File: pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/BrokersBase.java [hunk 2]
```diff
@@ -113,10 +121,12 @@
...
-    @ApiResponses(value = { @ApiResponse(code = 204, message = "Service configuration updated successfully"),
-            @ApiResponse(code = 403, message = "You don't have admin permission to update service-configuration"),
-            @ApiResponse(code = 404, message = "Configuration not found"),
-            @ApiResponse(code = 412, message = "Configuration can't be updated dynamically") })
+    @ApiResponses(value = {
+        @ApiResponse(code = 204, message = "Service configuration updated successfully"),
+        @ApiResponse(code = 403, message = "You don't have admin permission to update service-configuration"),
+        @ApiResponse(code = 404, message = "Configuration not found"),
+        @ApiResponse(code = 412, message = "Invalid dynamic-config value"),
+        @ApiResponse(code = 500, message = "Service internal error")})
```
Comment (sijie) [inline pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/BrokersBase.java]: Internal server error?

Comment (zymap) [reply pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/BrokersBase.java]: yeah, updated

**Pull request merged**